### ISSUE TITLE
Permission Denied when Authenticating Via Ping Identity

### ISSUE THREAD

**Issue by clutchbeyers** [RedHatGov/workshop-dashboard#7]
I attempted a oc new-app of the dashboard to a namespace I own (I am also a cluster-admin across the entire cluster), and when I authenticate via ping identity from the route url to the dashboard it gets a "Access forbidden" even though I have cluster-admin. Curious if this is something with the oauth of ping identity that is causing this or not.

**Comment by clutchbeyers** [comment]
@jharmison-redhat nice to meet you the other day

**Comment by jharmison-redhat** [comment]
The README was never updated based on the forks - can you make sure you've updated the template URL to the one in this repo?

**Comment by clutchbeyers** [comment]
Yep, updated the command to 'oc new-app https://raw.githubusercontent.com/RedHatGov/workshop-dashboard/develop/templates/production.json' and still getting the Access forbidden after ping auth

**Comment by jharmison-redhat** [comment]
Alrighty - here's the one that was used in our lab this week. Apologies for pinging you around - I don't have an OCP cluster up at the moment.

https://raw.githubusercontent.com/redhat-cop/agnosticd/development/ansible/roles/ocp4-workload-workshop-admin-storage/files/production-cluster-admin.json

**Comment by jharmison-redhat** [comment]
You can see the Ansible tasklist that leveraged the template here:
https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/ocp4-workload-workshop-admin-storage/tasks/workload.yml

**Comment by jharmison-redhat** [comment]
One more thing to ping, their labguide content:
https://github.com/OpenShiftDemos/openshift-ops-workshops/tree/ocp4-prod

**Comment by clutchbeyers** [comment]
No worries, thanks for the links. I will keep poking around to see if I can get this to work with our oauth

**Comment by clutchbeyers** [comment]
![Screen Shot 2023-03-14 at 1 59 30 PM](https://user-images.githubusercontent.com/9657860/225096103-a158ed1f-76e0-404d-8208-ba362b93f8b0.png)

**Comment by clutchbeyers** [comment]
`GET /favicon.ico 302 0.940 ms - 58
{"uri":"https://oauth-openshift.apps.<cluster>/oauth/authorize?response_type=code&client_id=system%3Aserviceaccount%3Acmb-test%3Adashboard-user&redirect_uri=https%3A%2F%2Fdashboard-cmb-test.apps.paas-rosa-dev.q42h.p1.openshiftapps.com%2Foauth_callback&scope=user%3Ainfo&state=a15d7884-0c54-4ac0-9002-18729011e08f","level":"debug","message":"authorization_uri"}

GET /oauth_handshake?next=%2Ffavicon.ico 302 1.360 ms - 356

GET / 302 0.759 ms - 94
{"uri":"https://oauth-openshift.apps.<cluster>/oauth/authorize?response_type=code&client_id=system%3Aserviceaccount%3Acmb-test%3Adashboard-user&redirect_uri=https%3A%2F%2Fdashboard-cmb-test.apps.<cluster>%2Foauth_callback&scope=user%3Ainfo&state=86850a0b-0875-40f0-a3bc-7cf317b0d38f","level":"debug","message":"authorization_uri"}

GET /oauth_handshake?next=%2F 302 1.144 ms - 744
{"options":{"redirect_uri":"https://dashboard-cmb-test.<cluster>/oauth_callback","scope":"user:info","code":"sha256~<token removed>"},"level":"debug","message":"token_options"}
{"result":{"access_token":"sha256~<token removed>","expires_in":86400,"scope":"user:info","token_type":"Bearer"},"level":"debug","message":"auth_result"}
{"result":{"token":{"access_token":"<token removed>","expires_in":86400,"scope":"user:info","token_type":"Bearer","expires_at":"2023-03-15T17:58:54.865Z"}},"level":"debug","message":"token_result"}
{"users":[],"level":"info","message":"OpenShift admin users"}
{"name":"289138","level":"info","message":"OpenShift user name"}
{"name":"289138","level":"info","message":"User forbidden access"}

GET /oauth_callback?code=sha256~_<token removed>&state=86850a0b-0875-40f0-a3bc-7cf317b0d38f 403 209.905 ms - 18

GET /favicon.ico 302 0.647 ms - 58
{"uri":"https://oauth-openshift.apps.<cluster>/oauth/authorize?response_type=code&client_id=system%3Aserviceaccount%3Acmb-test%3Adashboard-user&redirect_uri=https%3A%2F%2Fdashboard-cmb-test.<cluster>%2Foauth_callback&scope=user%3Ainfo&state=9c8627a3-4208-45e1-8f4b-2fa978a680ae","level":"debug","message":"authorization_uri"}

GET /oauth_handshake?next=%2Ffavicon.ico 302 0.810 ms - 356`

**Comment by jharmison-redhat** [comment]
Users being `[]` strikes me as the problem there, lol. I'm not honestly sure what the lookup it's doing against OAuth to get that is, so yes - may have to mess with it a bit to work w/ your OAuth setup

**Comment by clutchbeyers** [comment]
@jharmison-redhat Hey there! Ever have a chance to take a peak at the oauth issues for deploying to rosa-managed clusters? We are at the point now where it would be super handy to have these workshop dashboards for some kubernetes custom training! Thanks again!
